pycombo/pyCombo.py

```diff
@@ -44,15 +45,24 @@ def execute(
     num_split_attempts : int, default 0
         Number of split attempts. If 0, autoadjust this number automatically.
     fixed_split_step : int, default 0
-        Step number to apply predifined split. If 0, use only random splits,
+        Step number to apply predefined split. If 0, use only random splits,
         if >0 sets up the usage of 6 fixed type splits on every fixed_split_step.
-    random_seedd : int, default 0
+    treat_as_modularity : bool, default False
```

Comment:
Do I understand correctly that using those precomputed modularities will speed up performance :

Reply:
Even if so, I believe it will be a negligible improvement.
The main purpose of this is for users to be able to provide their own custom modularity. The request to add this feature came from Stan, as he wanted to treat loop edges differently.

Reply:
ah ok, perhaps we could explain that in the docstring